src/main/java/calculate/domain/Number.java

```diff
@@ -3,30 +3,32 @@
 import java.util.Objects;
 
 public class Number {
+
     private static final String NUMBER_FORMAT_ERROR_MESSAGE = "문자열 계산기에는 음수가 입력될 수 없습니다.";
 
     private final int value;
 
     public Number(final String value) {
-        this.value = isValidate(makeNumber(value));
+        this.value = makeNumber(value);
+        validateNonNegativeValue(this.value);
```

Comment:
```suggestion
        validateNonNegativeValue(this.value);
        this.value = makeNumber(value);
```
유효성을 먼저 검사하는 것도 좋은 방법일 것 같네요 :)

Reply:
넵..!